### PR TITLE
skaffold to deploy to local k8s

### LINKED ISSUE #32: twirp: k8s migrate grpc example
It would be worth to try using https://skaffold.dev to streamline dev process.

### PR DESCRIPTION
**Pull request by karol-kokoszka**
Fixes #32 

It adds support for skaffold that manages the deployment to minikube.

- Docker image build time is quite long as it generates the code with make gen (TBD)

Usage:
- `make k8s-setup-local`
- `make k8s-local`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,6 +16,20 @@ install-dependencies:
 	go install github.com/envoyproxy/protoc-gen-validate@latest
 	go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
 
+check-skaffold:
+	@which skaffold>/dev/null 2>&1 || (echo "you must install skaffold first -> https://skaffold.dev/docs/install/" && exit 1)
+
+check-minikube:
+	@which minikube>/dev/null 2>&1 || (echo "you must install minikube first -> https://minikube.sigs.k8s.io/docs/start/" && exit 1)
```
Comment (mmatczuk) [inline Makefile:23]: Can we actually install them from source https://github.com/GoogleContainerTools/skaffold/tree/main/cmd/skaffold

Comment (mmatczuk) [reply Makefile:23]: minikube you can assume it's installed

**Comment by mmatczuk** [comment]
It shall use local builder https://skaffold.dev/docs/pipeline-stages/builders/custom/#custom-build-script-locally

**Comment by mmatczuk** [comment]
Integrate with https://github.com/mmatczuk/scylla-cloud-poc/issues/56

File: Makefile (hunk 1 above)
Comment (karol-kokoszka) [reply Makefile:23]: to install from sources there is a need to clone and `make install`.
Does it make sense ?
Simple one liner doesn't work:
```
➜  skaffold git:(main) ✗ go install github.com/GoogleContainerTools/skaffold@latest
go install github.com/GoogleContainerTools/skaffold@latest: module github.com/GoogleContainerTools/skaffold@latest found (v1.36.1), but does not contain package github.com/GoogleContainerTools/skaffold
```

`make install` installs skaffold in GOBIN

**Comment by karol-kokoszka** [comment]
Last but not least - api server starts with `--book-addr http://book-svc:50051` and all containers have `curl` available.
To test it, you can shell into the container and execute for example:
`curl api-svc:8081/v1/books/1`
It will reach API service and route it to one of the pods (round-robin) and api-svc will connect to `book-svc` which will choose one of the pods as well (round-robin) again.

**Pull request merged**